docs/api/index.md

```diff
@@ -857,6 +857,17 @@ Route record that can be provided by the user when adding routes via the [`route
 
 - **See Also**: [Meta fields](../guide/advanced/meta.md)
 
+:::tip
+If you want to use a functional component, make sure to add a `displayName` to the component.
+
+For example:
+```js
+const Functional = () => h('div', 'functional')
+Functional.displayName = 'Functional'  // add a "displayName" to the component.
+const router = createRouter({ path: '/foo', component: Functional })
```

Comment:
can you replace this with a `const routes = [{ ... }` like in the rest of the docs?